setup.py

```diff
@@ -143,30 +129,14 @@ def run(self):
         "Operating System :: MacOS :: MacOS X",
         "Operating System :: POSIX :: Linux",
     ],
-    packages=find_packages(
-        where=".", exclude=["benchmarks", "benchmarks.*", "*_test.py"]
-    ),
+    packages=find_packages(where=".", exclude=["*_test.py"]),
     install_requires=install_requires,
     tests_require=tests_require,
     extras_require={
         "test": tests_require,
-        "gpu": gpu_require,
         "docs": docs_require,
-        "benchmarks": benchmarks_require,
-        "examples": examples_require,
-        "dev": dev_require
-        + tests_require
-        + benchmarks_require
-        + docs_require
-        + examples_require,
+        "dev": dev_require + tests_require + docs_require,
     },
     cmdclass={"develop": CustomDevelopCommand},
-    entry_points={
-        "console_scripts": [
-            "coordinator=xain_fl.cproto.cli:main",
-            "train_remote=benchmarks.train_remote:main",
-            "pull_results=xain_fl.ops.__main__:download",
-            "aggregate=benchmarks.aggregate:main",
-        ]
-    },
+    entry_points={"console_scripts": ["coordinator=xain_fl.cproto.cli:main"]},
```

Comment:
i think it's my fault, it's `xain_fl.cli:main` now (please double-check)

Reply:
Yeah, also my fault. I saw it and forgot to change it. Resolved.